Fixes #8

vdm-dev opened the pull request.

vdm-dev committed:
Fixes #8

sambertrdk commented on python/theme_Motoman.py:
```diff
@@ -46,18 +46,19 @@
-repo_match(repository, match, name_control, "control")
+# Don't need this now, because of match = r'\b[BE]?C\d{5}' above ^^^
+#match = r'\b[BE]?C\d{5}='#REGEX for position constants ex C00002=418.570,147.321,251.179,-180.00,23.13,0.00
+#repo_match(repository, match, name_control, "control")
```
You can remove the commented code above

sambertrdk commented on python/theme_Motoman.py:
```diff
@@ -46,18 +46,19 @@
 
 
-match = r'\WP\d+' #regex for position use ex P00095
+match = r'\bP\d{5}' #regex for position use ex P00095
```
The header of this file cites ABB and not Yaskawa, if you can fix it in this PR.

vdm-dev committed:
Code review: fixed Yaskawa Motoman file description
Code review: removed unnecessary comments
Updated VSIX package (version number is unchanged)
